JustBuyIt.lua

```diff
@@ -102,12 +103,14 @@ f:SetScript("OnEvent", function(self, eventName, ...)
         resetQuantity = false
         text:SetText("Failed")
         buyButton:SetEnabled(true)
+        refreshListings()
     end
 
     if (eventName == "COMMODITY_PURCHASE_SUCCEEDED") then
         listenToCommoditiesEvents = false
         text:SetText("Success")
         buyButton:SetEnabled(true)
+        refreshListings()
```

Comment:
Is this working for you? It never finishes refreshing for me

Reply:
Confirming that this works for me.

- Downloaded the post-change file from github
- Replaced the contents of `JustBuyIt.lua` with the file
- `/reload` in the WOW client
- Open AH to a commodity (Orbinid, as example)
- Entered 1g and "Just Buy It"
- "Too expensive" result and the item listing was refreshed

Reply:
It seems to be working when it is too expensive, but not on successful purchases

Reply:
Odd, this does seem to work for me. I'm using the same set of instructions above, but instead of entering a price that's too low I enter one that will likely succeed (i.e. 60 for orbinid at this time) and when the buy succeeds the item list is refreshed.

Reply:
It turned out to be an old modified version of Point Blank Sniper that was causing it. Thanks!